ceph.spec.in

```diff
@@ -820,6 +820,23 @@ mkdir -p %{_localstatedir}/run/ceph/
 %config %{_sysconfdir}/bash_completion.d/rbd
 %config(noreplace) %{_sysconfdir}/ceph/rbdmap
 %{python_sitelib}/ceph_argparse.py*
+%if 0%{?rhel} >= 7 || 0%{?fedora}
+/usr/lib/udev/rules.d/50-rbd.rules
+%else
```

Comment:
ah yes, but this from the first commit in this PR, which restores Ken Dreyer's code from the upstream spec file.